### PR TITLE
Copy-edit of README.

### PR DESCRIPTION
**Pull request by Julian-O**
Started with removing gender-specific language, and then went for a minor tidy-up of the rest of the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ Buildozer
 
 Buildozer is a tool for creating application packages easily.
 
-The goal is to have one "buildozer.spec" file in your app directory, describing
-your application requirements and settings such as title, icon, included modules
-etc. Buildozer will use that spec to create a package for Android, iOS, Windows,
-OSX and/or Linux.
+The goal is to have one "buildozer.spec" file in your app directory that describes
+your application requirements and settings - for example, title, icon and  included modules.
```
Comment (tshirtman) [inline README.md:12]:
```suggestion
your application requirements and settings - for example, title, icon and included modules.
```

File: docs/source/specifications.rst [hunk 2]
```diff
@@ -105,7 +105,7 @@ Section [app]
     garden_requirements = graph
 
   Please note that if it doesn't work, it might be because of the garden
-  package itself. Refer to the author of the package if he already tested
+  package itself. Refer to the author of the package if they already tested
```
Comment (tshirtman) [inline docs/source/specifications.rst:108]: i like the singular they, but here there could even be multiple authors, so i think we could even use a plural form.
```suggestion
  package itself. Refer to the author(s) of the package if they already tested
```

**Comment by Julian-O** [comment]
Updated with @tshirtman's suggestions. Thank you for the comments.

**Comment by tshirtman** [comment]
wait, the diff i looked at was just one file, and i saw there were a lot more diffs after validation, seems something went wrong with branching, this should probably be rebased.

**Comment by akshayaurora** [comment]
Don't mix document changes with code. This is not mergeable in it's current state. I would suggest to do a new pr with a clean base and to separate out doc prs from code prs.

**Pull request closed without merging**